msgpack-lite/README.md

```diff
@@ -2,7 +2,7 @@
 
 [](dependency)
 ```clojure
-[cljsjs/msgpack-lite "0.1.16-0"] ;; latest release
+[cljsjs/msgpack-lite "0.1.26"] ;; latest release
```

Comment:
Looks good otherwise, but this should be `0.1.26-0`, i.e. the `-0` is missing. Could you fix it?